README.md

```diff
@@ -126,15 +127,17 @@ frontend as specified above. In addition, it:
 Once you've chosen a setup that meets the above constraints, actual
 configuration is fairly straightforward:
 
-  1. `git clone http://github.com/ampproject/amp-packager && cd amp-packager && go build bin/amppkg.go`
+  1. `git clone http://github.com/ampproject/amp-packager && cd amp-packager && go build cmd/amppkg/main.go`
```

Comment:
this is now simply
`go get -v http://github.com/ampproject/amp-packager/cmd/amppkg`

Reply:
Done, thanks.